tests/docker/Dockerfile

```diff
@@ -49,12 +49,16 @@ RUN mkdir -p "/opt/kafka-0.10.1.0" && chmod a+rw /opt/kafka-0.10.1.0 && curl -s
 RUN mkdir -p "/opt/kafka-0.10.1.1" && chmod a+rw /opt/kafka-0.10.1.1 && curl -s "$KAFKA_MIRROR/kafka_2.11-0.10.1.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-0.10.1.1"
 RUN mkdir -p "/opt/kafka-0.10.2.0" && chmod a+rw /opt/kafka-0.10.2.0 && curl -s "$KAFKA_MIRROR/kafka_2.11-0.10.2.0.tgz" | tar xz --strip-components=1 -C "/opt/kafka-0.10.2.0"
 RUN mkdir -p "/opt/kafka-0.10.2.1" && chmod a+rw /opt/kafka-0.10.2.1 && curl -s "$KAFKA_MIRROR/kafka_2.11-0.10.2.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-0.10.2.1"
+RUN mkdir -p "/opt/kafka-0.10.2.2" && chmod a+rw /opt/kafka-0.10.2.1 && curl -s "$KAFKA_MIRROR/kafka_2.11-0.10.2.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-0.10.2.2"
```

Comment:
This didn't occur to me in the other PRs, but do we need to keep downloading the other bugfix versions (0.10.2.0 and 0.10.2.1), or do we only use the newest one?

Reply:
We will only download the latest one for each major.minor version, since for each version we'd always recommend users to to use the latest bug-fix one, and hence testing wise we will only check for that version as well.

Reply:
Ok, then we can remove the unused downloads from this Dockerfile and speed up the tests.

Reply:
Ah, I see this is actually what we already do in the Vagrant config. I guess someone started a bad precedent in the Docker config at some point.